.github/workflows/packages.yml

```diff
@@ -76,9 +77,10 @@ jobs:
         uses: docker/setup-qemu-action@v3
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.21.2
+        uses: pypa/cibuildwheel@v2.23.3
         env:
           CIBW_BUILD: ${{matrix.pyver}}-*
+          CIBW_ENABLE: cpython-freethreading
```

Comment:
This doesn't bother the non-freethreading build, it just adds the option to build `cp313t-`, right?

Reply:
That's correct, yes.